cmd/describe/cluster/cmd.go

```diff
@@ -335,6 +286,74 @@ func run(cmd *cobra.Command, argv []string) {
 	fmt.Println()
 }
 
+func controlPlaneConfig(cluster *cmv1.Cluster) string {
+	if cluster.Hypershift().Enabled() {
+		return "Red Hat hosted"
+	}
+	return "Customer hosted"
+}
+
+func clusterInfraConfig(cluster *cmv1.Cluster, clusterKey string, r *rosa.Runtime) string {
+	var infraConfig string
+	if cluster.Hypershift().Enabled() {
+		infraConfig = fmt.Sprintf(""+
+			"Node Pools:\n"+
+			" - Compute:                 %d\n",
+			cluster.Nodes().Compute(),
```

Comment:
This variable will not hold the entire NodePool count, though?

Reply:
These changes are from the other PR: https://github.com/openshift/rosa/pull/837
I think I mistakenly created a branch off the above PR :/